docs/docsite/rst/porting_guides/porting_guide_2.10.rst

```diff
@@ -62,7 +62,7 @@ Noteworthy module changes
 * :ref:`vmware_host_ntp <vmware_host_ntp_module>` now returns ``host_ntp_status`` instead of Ansible internal key ``results``.
 * :ref:`vmware_host_service_manager <vmware_host_service_manager_module>` now returns ``host_service_status`` instead of Ansible internal key ``results``.
 * :ref:`vmware_tag <vmware_tag_module>` now returns ``tag_status`` instead of Ansible internal key ``results``.
-
```

Comment:
Please keep two lines before the next section.

Reply:
@felixfontein done